File: Arnav Sharma/Assignment-1/unique.java
package com.array;

public class unique {
    public static void main (String[] args)
    {
        int arr[] = { 10, 3, 5, 3, 9, 22, 4, 3, 1, 5, 6 };

        int len = arr.length;

        int[] newArr = new int[20];

        int index = 0;
        for (int i = 0; i < len; i++)
        {
            int flag = 0;
            for (int j = 0; j < i; j++){
                if (arr[i] == arr[j]){
                    flag = 1;
                    break;
                }
            }

            if (flag == 0){
                newArr[index] = arr[i];
                index++;
            }
        }
        for(int i = 0; i < index; i++)
            System.out.print( newArr[i] + " ");
    }
}

File: akashmartule/Arrays/unique.cpp
#include<iostream>>
using namespace std;
int main()
{
    int arr[50],freq[50];
    int i,j,size,count;

    cout<<"number of elements in array :";
    cin>>size;
    cout<<"enter elements in array:  ";
    for(i=0;i<size;i++){
       cin>>arr[i];
       freq[i]=-1;           
    }
    /* Find frequency of each element */
    for(i=0;i<size;i++);{
    count=1;
       
      for(j=i+1; j<size; j++)
        {
            if(arr[i] == arr[j])
            {
                count++;
                freq[j] = 0;
            }

        }
        if(freq[i] != 0)
        {
            freq[i] = count;
        }
    }
         /* Print all unique elements of array */

         for(i=0;i<size;i++){
             if(freq[i]==1){
               cout<<arr[i]<<" ";
             }
         }
         return 0;
         
}